kombu/pidbox.py

```diff
@@ -227,7 +243,18 @@ def get_queue(self, hostname):
         return Queue('%s.%s.pidbox' % (hostname, self.namespace),
                      exchange=self.exchange,
                      durable=False,
-                     auto_delete=True)
+                     auto_delete=True,
+                     queue_arguments=self._get_queue_arguments())
+
+    def _get_queue_arguments(self, ttl=None, expires=None):
```

Comment:
As this this is very related to the queue, my 1st sight is that it belongs to the Queue class. 

What about a staticmethod or a utils package? The proposed change seems less readable and too coupled for me :/